src/utils.rs

```diff
@@ -1291,7 +1291,7 @@ pub fn inv_parametrize3d<T: FloatLike>(
         }
         ParameterizationMapping::Linear => {
             let b = Into::<T>::into(settings.parameterization.b);
-            jac /= <T as num_traits::Float>::powi(e_cm * b + k_r, 2) / e_cm / b;
+            jac /= <T as num::traits::Float>::powi(e_cm * b + k_r, 2) / e_cm / b;
```

Comment:
How come we no longer need our custom `num_traits`?

Reply:
@lcnbr Also the `examples` folder should be cleaned up a bit.
Also because our gammaloop project is a hybrid of python and rust, I would create two folders there, `rust` and `python`, and then within the `rust` one organise the example with subfolders for various categories, e.g. `tensors`, in your case.